pkg/apis/operator/v1alpha1/tektonresult_types.go

```diff
@@ -82,10 +82,14 @@ type ResultsAPIProperties struct {
 	S3AccessKeyID         string `json:"s3_access_key_id,omitempty"`
 	S3SecretAccessKey     string `json:"s3_secret_access_key,omitempty"`
 	S3MultiPartSize       int64  `json:"s3_multi_part_size,omitempty"`
-	LoggingPVCName        string `json:"logging_pvc_name"`
+	LoggingPVCName        string `json:"logging_pvc_name,omitempty"`
+	GcsBucketName         string `json:"gcs_bucket_name,omitempty"`
+	StorageEmulatorHost   string `json:"storage_emulator_host,omitempty"`
```

Comment:
I see you have not used this property => `StorageEmulatorHost` in this PR.

Reply:
This is used in configmap. Check the unittest.